vars/rapidastUtils.groovy

```diff
@@ -57,21 +57,21 @@ def prepareRapidastStages(String ServiceName, String PluginName, String ApiScann
 
         stage("Create Jira tickets for alerts") {
             //Typecast Jira from String to Hashmap for easier usage
-            Jira = StringToMap(Jira)
-            if (Jira) {
+            jiraMap = StringToMap(Jira)
+            if (jiraMap) {
                 catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
                     def sarif_file = findFiles(glob: "results/${ServiceName}/**/zap/zap-report.sarif.json")[0]
                     sh "git -c http.sslVerify=false clone https://gitlab.cee.redhat.com/fcanogab/sariftojira"
                     dir("sariftojira") {
                         withCredentials([string(credentialsId: 'JIRA_TOKEN', variable: 'JIRA_TOKEN')]) {
                             sh "export JIRA_TOKEN=${JIRA_TOKEN}"
```

Comment:
```
Warning: A secret was passed to "sh" using Groovy String interpolation, which is insecure.
		 Affected argument(s) used the following variable(s): [JIRA_TOKEN]
		 See https://jenkins.io/redirect/groovy-string-interpolation for details.
+ export JIRA_TOKEN=****
```

We need to use here single quotes, [see](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#interpolation-of-sensitive-environment-variables)

Reply:
Testing it